document/CoreFormat.mdk

```diff
@@ -1432,12 +1566,61 @@ A C++ analysis tool might emit the `name` property of a function as the "decorat
 which encodes the function signature in a manner that is compiler-dependent and not easily readable.
 ~
 
+
+If the `logicalLocationComponent` object occurs as the only member of an array within
+the `run.logicalLocations` object (&sect;[#run-logicalLocations]),
+and if the `name` property would be equal to the name of the corresponding property, then the `name` property may be absent.
+
+~ IsoNumberedExample { @IsoNumberedExample=1 }
+In this example, the array of `logicalLocationComponent` objects has only one element, and `name` is omitted.
+```
+"logicalLocations": {
+  "functionF": [
+    {
+      "kind": "function"
+    }
+  ]
+}
+```
+~
+
+~ IsoNumberedExample
+In this example, the array of `logicalLocationComponent` objects has only one element, and `name` is equal to the property name.
+```
+"logicalLocations": {
+  "functionF": [
+    {
+      "pathFromParent": "functionF",
+      "kind": "function"
+    }
+  ]
+}
+```
+~
+
+~ IsoNumberedExample
+In this example, the array of `logicalLocationComponent` objects has only one element,
+but `name` is not equal to the property name, so it cannot be omitted.
+```
+"logicalLocations": {
+  "functionF-0": [
+    {
+      "pathFromParent": "functionF",
+      "kind": "function"
+    }
+  ]
+}
+```
+~
+
 ### `kind` property { #logicalLocationComponent-kind }
 
 A `logicalLocationComponent` object should contain a property named `kind`, whose value shall be one of the following strings,
 if any of those strings accurately describes the construct identified by this object:
 
-*	`method`
```

Comment:
You can add method as far as I'm concerned, since we saw it in Android Studio. :)